x-pack/plugins/apm/public/components/app/tutorials/commands/node.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+import {
+  serviceNameHint,
+  secretTokenHint,
+  serverUrlHint,
+  serviceEnvironmentHint,
+  apiKeyHint,
+} from './shared_hints';
+
+export const nodeVariables = {
+  apmServiceName: 'serviceName',
+  secretToken: 'secretToken',
+  apiKey: 'apiKey',
+  apmServerUrl: 'serverUrl',
+  apmEnvironment: 'environment',
+};
+
+export const nodeHighlightLang = 'js';
+
+const nodeServiceNameHint = i18n.translate(
+  'xpack.apm.tutorial.nodeClient.createConfig.commands.serviceName',
+  {
+    defaultMessage: 'Overrides the service name in package.json.',
+  }
+);
```

Comment:
Localization is nice but I think it's more important to have something that's really simple to maintain and update.

Reply:
Removed and moved it to Code Comment